src/removeElements.js

```diff
@@ -1,23 +1,28 @@
 
 export function removeData() {
     const generalInfoDiv = document.getElementsByClassName("general-info-container").item(0);
-    generalInfoDiv.remove();
+    if (generalInfoDiv) generalInfoDiv.remove();
```

Comment:
I guess here you are doing a safe navigation, am I right?

If so, you can use: *generalInfoDiv?.remove()*

You can also do this for all this cases.